ci/release/changelogs/next.md

```diff
@@ -1,21 +1,25 @@
+D2 now has an official playground site: [https://play.d2lang.com](https://play.d2lang.com). It loads and runs fast, works on all the major browsers, tested on desktop and mobile on a variety of devices, and is the easiest way to get started with D2 and share diagrams. The playground is all open source ([https://github.com/terrastruct/d2-playground](https://github.com/terrastruct/d2-playground)), and we'd love to hear your feedback and feature requests.
+
+Windows users, the install experience just got a whole lot better. Making D2 accessible and easy to use continues to be a priority for us, and with this release, we added an MSI installer for Windows, so that installs are just a few clicks. An official Docker image has also been added.
```

Comment:
```suggestion
Windows users, the install experience just got a whole lot better. Making D2 accessible and easy to use continues to be a priority for us. With this release, we added an MSI installer for Windows, so that installs are just a few clicks. An official Docker image has also been added.
```